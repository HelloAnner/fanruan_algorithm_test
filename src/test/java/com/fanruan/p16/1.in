abc+!@#$%^&*(acb)[234]{123++--}\//\=qwretqretrewcsv