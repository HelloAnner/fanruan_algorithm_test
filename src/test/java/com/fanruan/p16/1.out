abc__________________(acb)[234]{123____--}__________qwretqretrewcsv